units/Legion/legcom.lua

```diff
@@ -97,11 +100,6 @@ return {
 		customparams = {
 			unitgroup = 'builder',
 			combatradius = 600,
-			evolution_announcement = "Legion commanders have upgraded to level 2",
-			evolution_announcement_size = 18.5,
-			evolution_target = "legcomlvl2",
-			evolution_condition = "timer",
-			evolution_timer = 600,
```

Comment:
this would have broken modoptions for enabling evocom, that's been fixed pre-emptively in https://github.com/beyond-all-reason/Beyond-All-Reason/pull/3131/commits